.github/workflows/tools.yml

```diff
@@ -23,6 +23,9 @@ jobs:
                 echo "NEW_VERSION=$NEW_VERSION" >> $GITHUB_ENV
                 ./update-eslint.sh
               fi
+          - id: corepack
+            run: |
+              make corepack-update
```

Comment:
```suggestion
              make corepack-update
              echo "NEW_VERSION=$(node deps/corepack/dist/corepack.js --version)" >> $GITHUB_ENV
```

Reply:
Why set `NEW_VERSION` if it's not being used?

Reply:
I wondered the same as @mscdex 😄 But good point, I didn't see we use this variable in a subsequent step.